app/src/androidTest/java/com/github/se/orator/ui/settings/SettingsScreenTest.kt

```diff
@@ -52,21 +52,16 @@ class SettingsScreenTest {
     }
 
     // Test that each setting button exists and is clickable
-    val settingsTags =
-        listOf(
-            "account_management",
-            "storage_settings",
-            "permissions",
-            "theme",
-            "invite_friends",
-            "notifications",
-            "rate_on_the_app_store",
-            "about")
+    val settingsTags = listOf("theme", "about")
 
     settingsTags.forEach { tag ->
       composeTestRule.onNodeWithTag(tag).assertExists()
       composeTestRule.onNodeWithTag(tag).performClick()
     }
+
+    // Handle the permission button on its own as clicking it will go to the device settings
+    composeTestRule.onNodeWithTag("permissions").assertExists()
+    composeTestRule.onNodeWithTag("permissions").assertHasClickAction()
```

Comment:
Great workaround to the permissions screen pop up! 
Maybe using .assertHasClickAction on every tag instead of having two separate blocks of code would be even better

Reply:
Hey, I did this because clicking on the permission button leaves the app, and coming back to it is a problem. This is explained by the comment. The other button's click action should still be tested though.

Reply:
Yes I'm agreeing with you and saying it's a clever way to test the button even though a screen pops up.
What I was saying is maybe use this on other buttons too like  composeTestRule.onNodeWithTag("about").assertHasClickAction() since for now we are just clicking them

Reply:
Done.